test/record_test.dart

```diff
@@ -15,6 +16,7 @@ import 'utils.dart';
 
 void main() {
   FileSystem fs = new LocalFileSystem();
+  String newline = Platform.isWindows ? '\r\n' : '\n';
```

Comment:
FYI, I just created https://github.com/google/platform.dart/issues/1 -- maybe add a comment referencing it here so that when it's implemented, we can switch to use it here.